BusinessLayer/LoggerService/LoggerService.cpp.bak

```diff
@@ -0,0 +1,174 @@
+#include <QDebug>
```

Comment:
Actually, `.orig` files are the ones that come from AStyle....this `.bak` file must have been from somewhere else then